docs/markdown/Cross-compilation.md

```diff
@@ -7,14 +7,17 @@ short-description: Setting up cross-compilation
 Meson has full support for cross compilation. Since cross compiling is
 more complicated than native building, let's first go over some
 nomenclature. The three most important definitions are traditionally
-called *build*, *host* and *target*. This is confusing because those
+called *build*, *host* and *target*. This is confusing because those70
```

Comment:
Typo: those70

Reply:
Oops sorry! Must have down that when changing the max characters per line haha.